lib/_stream_duplex.js

```diff
@@ -21,6 +14,13 @@ var objectKeys = Object.keys || function (obj) {
 /*</replacement>*/
 
 
+module.exports = Duplex;
```

Comment:
since the tests don't have an export I switched the objectKeys replacement to hook onto the `'use strict';` statement which is now in every file.